src/Control/Effect/Lift.hs

```diff
@@ -1,18 +1,25 @@
-{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE FlexibleContexts, MultiParamTypeClasses #-}
 module Control.Effect.Lift
 ( Lift(..)
+, lift
 , runM
 , LiftC(..)
 ) where
 
 import Control.Effect.Carrier
+import Control.Effect.Sum
 import Control.Effect.Internal
 import Control.Effect.Lift.Internal
 
 -- | Extract a 'Lift'ed 'Monad'ic action from an effectful computation.
 runM :: Monad m => Eff (LiftC m) a -> m a
 runM = runLiftC . interpret
 
+-- | Given a @Lift n@ constraint in a signature carried by @m@, use 'lift' to
+-- promote arbitrary actions of type @n a@ to @m a@.
+lift :: (Member (Lift n) sig, Carrier sig m, Functor n, Applicative m) => n a -> m a
+lift = send . Lift . fmap pure
```

Comment:
What do you think about `sendM`, by analogy with `runM`, above?

Reply:
I think this is a good name. I'll document its spiritual similarity with `lift` from MTL.